xbmc/cores/VideoPlayer/DVDCodecs/Video/AddonVideoCodec.cpp

```diff
@@ -29,8 +29,6 @@
 
 using namespace kodi::addon;
 
-#define ALIGN(value, alignment) (((value)+(alignment-1))&~(alignment-1))
-
```

Comment:
@FernetMenta is it okay for you to change this here or do you want me to PR it to vpupdates?